components/03-organisms/_grid-mixins.scss

```diff
@@ -0,0 +1,58 @@
+@use '../00-tokens/tokens';
+@use '../01-atoms/atoms';
+
+@mixin base {
+  @include atoms.list-reset;
+
+  display: flex;
+  flex-flow: row wrap;
+  gap: var(--grid-gutter);
+  justify-content: center;
+
+  > * {
+    flex-basis: 100%;
+  }
+}
+
+@mixin primary {
+  --grid-gutter: var(--spacing-component-gutter);
+
+  @include base;
+
+  > * {
+    @media (min-width: tokens.$break-m) {
+      flex: 0 1 calc(50% - calc(var(--grid-gutter) / 2));
+    }
+
+    @media (min-width: tokens.$break-l) {
+      flex: 0 1 calc(33.33% - calc(var(--grid-gutter) * 2 / 3));
+    }
+  }
+}
+
+@mixin secondary($media-only: false) {
```

Comment:
Logic here looks good and can be merged but I would consider making this more generic if you are inclined. Instead of passing "what" the grid is going to be used for you could pass the breakpoint where you want 2 columns to kick-in. So `@include grid.secondary(true);` could be written as `@include grid.secondary('break-s');` <-- I think this is easier to discern what the argument passing is doing at implementation. 

soooo 

```
@mixin secondary($breakpoint: null) {
...
> * {
    // Media-only grids are always two-up
    @if $breakpoint {
      // Grids with content (like the news cards) start one-up, and go to two-up
      // at this breakpoint.
      @media (min-width: tokens.$breakpoint) {
        flex: 0 1 calc(50% - calc(var(--grid-gutter) / 2));
      }
    } @else {
        flex: 0 1 calc(50% - calc(var(--grid-gutter) / 2));
    }
   ...
```

Reply:
I disagree here. While it would make it more flexible, the reason the news grids don't start at two-up is specifically because they have text in addition to the image. And the reason the "image-only" grids are always two-up - even at the smallest screen sizes - is specifically because they don't have any "extra information".

So, to allow someone to place news cards in a grid that start at two-up would break design requirements because the text area would be so narrow, you'd only get one word on a line (or they might even overlap the next card). We want to actively avoid that, and just giving a wide-open, use-case-less (if that's a word) variable makes that mistake easier to make.

So, I think the logic (and variable naming) for this use-case is correct as is:

> If there's no additional text to display with the image - it's always at least two-up

> If there is supplemental text shown below the image - be one-up until the browser is wide enough to support showing that textual content in a two-up display

Reply:
I see what you're saying. So is 
```
.media-grid__items {
  @include grid.secondary(true);
}
``` 
essentially the only instance this will be used given other grid items will contain text? For me, what prompted this feedback was the single use case for this mixin argument and the fact that the implementation `@include grid.secondary(true);` isn't clear what it's doing. This mixin is a bit on-the-rails to enforce design requirements which I think is fine but wanted to be clear on your thought process around it.

Reply:
I was on the fence about how to use the mixin. Since it's the first (and only) argument, we don't have to name it when we use it. But I went ahead and named it in the usage so that it's more clear as to what the argument is for. I'm going to merge this now, but if you think this could still be improved, let's do it in the "interactive grid" PR.

Here's what it looks like now, if you don't want to review the code:
```
.media-grid__items {
  @include grid.secondary($media-only: true);
}
```